test/e2e/framework/service_exports.go

```diff
@@ -0,0 +1,50 @@
+/*
+© 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package framework
+
+import (
+	"fmt"
+
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	mcsv1a1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+)
+
+func (f *Framework) CreateServiceExport(cluster ClusterIndex, name string) {
+	svcEx := mcsv1a1.ServiceExport{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: f.Namespace,
+		},
+	}
+	resourceServiceExport, gvr, err := ToUnstructuredResource(&svcEx, RestMappers[cluster])
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
You can create an Unstructured directly - you just need to set `Name` and `Namespace`. Then you don't need to copy `ToUnstructuredResource` et al from admiral and no need for `RestMappers`. 

```suggestion
	resourceServiceExport := &unstructured.Unstructured{}
	resourceServiceExport.SetName(name)
	resourceServiceExport.SetNamespace(f.Namespace)
	
	gvr, _ := schema.ParseResourceArg("serviceexports.v1alpha1.multicluster.x-k8s.io")
```

Reply:
This didn't work, got this error:
```
Message: "ServiceExport in version \"v1alpha1\" cannot be handled as a ServiceExport: unmarshalerDecoder: Object 'Kind' is missing in '{\"metadata\":{\"name\":\"test-svc-tcp-check-listener\",\"namespace\":\"e2e-tests-dataplane-bqpb9\"}}', error found in #10 byte of ...|e-bqpb9\"}}\n|..., bigger context ...|istener\",\"namespace\":\"e2e-tests-dataplane-bqpb9\"}}\n|...",
```

Reply:
OK. Then you can just do:

```
svcEx := mcsv1a1.ServiceExport{
    ObjectMeta: metav1.ObjectMeta{
	Name:      name,
	Namespace: f.Namespace,
    },
}

resourceServiceExport := &unstructured.Unstructured{}
Expect(scheme.Scheme.Convert(svcEx, resourceServiceExport, nil)).NotTo(HaveOccurred())
```

Reply:
Got this error:
```
    Unexpected error:
        <*errors.errorString | 0xc000620dd0>: {
            s: "unable to convert object type v1alpha1.ServiceExport to Unstructured, must be a runtime.Object",
        }
        unable to convert object type v1alpha1.ServiceExport to Unstructured, must be a runtime.Object
    occurred

```
It works if I manually set Kind and ApiVersion, so will go with that.